wormhole/file_transport.go

```diff
@@ -483,19 +497,28 @@ func (t *fileTransport) listen() error {
 	if t.disableListener {
 		return nil
 	}
-	switch t.relayURL.Scheme {
-	case "tcp":
-		l, err := net.Listen("tcp", ":0")
-		if err != nil {
-			return err
-		}
-
-		t.listener = l
-	case "ws", "wss":
-		t.listener = nil
-	default:
-		return fmt.Errorf("%w: '%s'", UnsupportedProtocolErr, t.relayURL.Scheme)
+	// always have tcp listener, otherwise app should run with --no-listen
+	l, err := net.Listen("tcp", ":0")
+	if err != nil {
+		return err
 	}
+	t.listener = l
+
+	// // This is in case we want completle disable direct tcp
```

Comment:
We don't need to keep the commented out code, please remove it

Reply:
removed